docs/getting-started.md

```diff
@@ -136,6 +136,11 @@ _will never work!_ This is because `user` is a promise object, not a data row fr
 User.findOne().then(user => {
   console.log(user.get('firstName'));
 });
+
+When your environment or transpiler supports [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) this will work:
+user = await User.findOne()
```

Comment:
I would mention that `await` can only be used inside `async` functions. This hint is clearly for people who don't know `async/await` yet.

just
```ts
async function requestHandler(req: Request, res: Response) {
    const user = await User.findOne();
    res.send(user);
}
```
or so would be perfect.